Nightscouter/SiteDetailViewController.swift

```diff
@@ -141,35 +141,30 @@ extension SiteDetailViewController {
     
     func updateSite(_ notification: Notification?) {
         print(">>> Entering \(#function) <<<")
-        self.updateData(forceUpdate: true)
+        self.updateData()
     }
     
-    func updateData(forceUpdate force: Bool = false) {
+    func updateData() {
         guard let site = self.site else { return }
+        UIApplication.shared.isNetworkActivityIndicatorVisible = true
         
-        if (site.updateNow || site.sgvs.isEmpty || force == true) {
```

Comment:
By removing the check for update now, etc... every time the user pushes into this view or pops back and forth we call the network even though it might not be the right time. Any specific reason why this was removed?